bears/natural_language/WriteGoodLintBear.py

```diff
@@ -42,20 +42,20 @@ def create_arguments(filename, file, config_file,
                          allow_extra_words: bool=True,
                          allow_cliche_phrases: bool=True):
         """
-        Using ``False`` will enable the check.
+        Using "False" will enable the check.
```

Comment:
This PR is only about human language words in the parameter descriptions, not `False` here, which is a technical term.